src/D2L.Security.OAuth2/Properties/AssemblyInfo.cs

```diff
@@ -32,9 +32,9 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion( "4.4.3.0" )]
-[assembly: AssemblyFileVersion( "4.4.3.0" )]
-[assembly: AssemblyInformationalVersion( "4.4.3.0" )]
+[assembly: AssemblyVersion( "4.5.0.0" )]
+[assembly: AssemblyFileVersion( "4.5.0.0" )]
+[assembly: AssemblyInformationalVersion( "4.5.0.0" )]

```

Comment:
It's preferable to not version bump within the PR because there are other changes going around... I'm going to bundle up a bunch of stuff and do a 5.0.0 shortly so could you remove this part?